examples/bert/tests/test_glue.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import shutil
+import tempfile
+from typing import Any
+
+from omegaconf import OmegaConf
+
+from ..glue import train
+
+class GlueDirContext(object):
+
+    def __enter__(self):
+        self.path = tempfile.mkdtemp() # type: ignore (reportUninitializedInstanceVariable)
+        return self.path
+
+    def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any):
+        del exc_type, exc_value, traceback # unused
+        shutil.rmtree(self.path)
+
+@pytest.mark.parametrize('model_name', ['mosaic_bert', 'hf_bert'])
+def test_glue_script(model_name: str):
+
+    with open('tests/smoketest_config_glue.yaml') as f:
```

Comment:
I'm not an expert at path stuff, but wondering if this requires the `pytest tests` be run from only the directory `examples/bert`? Would CI have trouble if running pytests from farther away?

Reply:
It might have trouble, but this is the pattern set across some of the other examples (e.g. cifar) when a "smoketest" config is part of the local `tests/`

Reply:
Taking a closer look at `scripts/test_subdirectory.py`, it look like the steps are to `cd` into the given subdirectory and run `python -m pytest tests`, so the way I have it set up should work :)

Reply:
Sweet! Thanks for explaining :)